docker/docker-compose.yml

```diff
@@ -93,6 +93,19 @@ services:
       volumes:
           - lnd_btc_data:/root/.lnd
 
+    alice:
+      extends: lnd
+      links:
+          - "btcd:blockchain"
+      volumes:
+          - alice_data:/root/.lnd
```

Comment:
This doesn't seem to work, as Alice an Bob will share the same folder, and won't be able to run at the same time.

Reply:
alice_data and bob_data are 2 different volumes and won't conflict.

Reply:
Okay, will need to do some more testing, running into some issues... Will report back!